### PR TITLE
standardize IRIs

### LINKED ISSUE #5: Standardize IRIs and prefLabels according to defined conventions
Conventions to be defined in https://github.com/semanticarts/gist/issues/1141.

### PR DESCRIPTION
**Pull request by philblackwood**
closes #5 

IRIs will look like gistd:_UnitOfMeasure_mile_per_hour  (infix followed by text using single underscore as delimiter)

### CODE REVIEW

File: Data/uomReferenceData.ttl [hunk 1]
```diff
@@ -6201,7 +6200,7 @@ gistd:_Aspect_work_function rdf:type gist:Aspect .
 gistd:_Aspect_work_function rdfs:seeAlso "http://en.wikipedia.org/wiki/Work_function"^^xsd:anyURI .
 gistd:_Aspect_work_function rdfs:seeAlso "http://www.iso.org/iso/catalogue_detail?csnumber=31897"^^xsd:anyURI .
 gistd:_Aspect_work_function skos:closeMatch <http://qudt.org/vocab/quantitykind/WorkFunction> .
-gistd:_Aspect_work_function skos:definition "from QUDT: \"Work Function\" is the energy difference between an electron at rest at infinity and an electron at a certain energy level. The minimum energy (usually measured in electronvolts) needed to remove an electron from a solid to a point immediately outside the solid surface (or energy needed to move an electron from the Fermi level into vacuum)." .
+gistd:_Aspect_work_function skos:definition "from QUDT: \"Work Function\" is the energy difference between an electron at rest at infinity and an electron at a certain energy level. The minimum energy (usually measured in electron_volts) needed to remove an electron from a solid to a point immediately outside the solid surface (or energy needed to move an electron from the Fermi level into vacuum)." .
```
Comment (kchastain703) [inline Data/uomReferenceData.ttl:6203]:
```suggestion
gistd:_Aspect_work_function skos:definition "from QUDT: \"Work Function\" is the energy difference between an electron at rest at infinity and an electron at a certain energy level. The minimum energy (usually measured in electronvolts) needed to remove an electron from a solid to a point immediately outside the solid surface (or energy needed to move an electron from the Fermi level into vacuum)." .
```

File: Data/uomReferenceData.ttl [hunk 2]
```diff
@@ -13914,22 +13913,22 @@ gistd:_UnitOfMeasure_electron_volt_second skos:altLabel "electron volt seconds"
...
-gistd:_UnitOfMeasure_electronvolt_per_meter rdf:type gist:UnitOfMeasure .
-gistd:_UnitOfMeasure_electronvolt_per_meter skos:altLabel "electronvolts per meter" .
-gistd:_UnitOfMeasure_electronvolt_per_meter skos:closeMatch <http://qudt.org/vocab/unit/EV-PER-M> .
-gistd:_UnitOfMeasure_electronvolt_per_meter skos:definition "from QUDT: unit electronvolt divided by the SI base unit metre" .
-gistd:_UnitOfMeasure_electronvolt_per_meter skos:prefLabel "electronvolt per meter" .
-gistd:_UnitOfMeasure_electronvolt_per_meter skos:scopeNote "1 electronvolt per meter = 0.0000000000000000001602176634 x kilogram meter per secondSquared" .
+gistd:_UnitOfMeasure_electron_volt_per_angstrom gist:conversionFactor "0.000000001602176634"^^xsd:decimal .
+gistd:_UnitOfMeasure_electron_volt_per_angstrom gist:isMemberOf gistd:_UnitGroup_force .
+gistd:_UnitOfMeasure_electron_volt_per_angstrom rdf:type gist:UnitOfMeasure .
+gistd:_UnitOfMeasure_electron_volt_per_angstrom skos:altLabel "electron_volts per angstrom" .
```
Comment (kchastain703) [inline Data/uomReferenceData.ttl:13919]:
```suggestion
gistd:_UnitOfMeasure_electronvolt_per_angstrom skos:altLabel "electronvolts per angstrom" .
```

For lines 13916-13931, it looks like "electronvolt" (no space) is the unit name, though "electron volt" is also acceptable. So the "electronvolt" -> "electron_volt" changes can be reverted if we want, OR if we want to keep "electron volt" throughout, the underscore is showing up in the prefLabels and altLabels and needs to be removed.

**Review by kchastain703 — CHANGES_REQUESTED** [review]
I noted below, it looks like "electronvolt" is correct for that unit, although "electron volt" is also acceptable. So for these, we should either
1) revert the changes back to "electronvolt" as it was before
or 2) if we keep "electron volt", the underscore also appears in the labels, and should be removed

**Comment by philblackwood** [comment]
I will create a separate issue for "electronvolt" vs. "electron volt".

**Pull request merged**